plugin.json

```diff
@@ -5,8 +5,8 @@
     "homepage_url": "https://github.com/mattermost/mattermost-plugin-demo",
     "support_url": "https://github.com/mattermost/mattermost-plugin-demo/issues",
     "icon_path": "assets/icon.svg",
-    "version": "0.5.0",
-    "min_server_version": "5.20.0",
+    "version": "0.6.0",
+    "min_server_version": "5.22.0",
```

Comment:
This is needed because of https://github.com/mattermost/mattermost-plugin-demo/pull/90